pennylane_lightning/core/src/utils/cuda_utils/cuGates_host.hpp

```diff
@@ -653,17 +654,25 @@ template <class CFP_t, class U = double>
 static auto getSingleExcitation(U angle) -> std::vector<CFP_t> {
     const U p2 = angle / 2;
     const CFP_t c{std::cos(p2), 0};
-    const CFP_t s{std::sin(p2), 0};
+    // TODO: To remove conditional compilation here in the future, current
+    // implementation will block the simultaneous installation of LGPU and
+    // cutensornet backends
```

Comment:
Wait, so we cannot install both packages together with this?

Reply:
So this is to be fixed in the next PR?

Reply:
Will do!